operator/api/v1alpha1/keptnevaluation_types.go

```diff
@@ -60,6 +60,7 @@ type KeptnEvaluationStatus struct {
 type EvaluationStatusItem struct {
 	Value  string            `json:"value"`
 	Status common.KeptnState `json:"status"`
+	Error  string            `json:"error,omitempty"`
```

Comment:
Would rename this to message instead of error

Reply:
maybe ErrMsg would be ok ?

Reply:
As this is a status item, it does not necessarily have to be an error, therefore I'd go for message ...